services/movie_session.py

```diff
@@ -0,0 +1,47 @@
+from django.utils import timezone
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> None:
+    if session_date:
+        date = timezone.datetime.strptime(session_date, "%Y-%m-%d").date()
```

Comment:
You don't need to use `datetime.strptime` here. `session date` already has an appropriate format.